test/utils/constants.ts

```diff
@@ -1,6 +1,7 @@
 import { constants } from 'ethers'
 
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
It seems in the past for zero address different constant was used. I'd prefer to stick with `AddressZero`. Let's remove this one.